chucks-pick3-ws/src/main/java/software/bytepushers/pick3/controllers/AccountTypeController.java

```diff
@@ -0,0 +1,47 @@
+package software.bytepushers.pick3.controllers;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import software.bytepushers.pick3.domain.AccountType;
+import software.bytepushers.pick3.repositories.AccountRepository;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import static software.bytepushers.pick3.config.security.SecurityConstants.ACCOUNT_TYPE_END_POINT;
+
+@RestController
+@RequestMapping(ACCOUNT_TYPE_END_POINT)
+public class AccountTypeController {
+
+    private final AccountRepository accountRepository;
```

Comment:
We should have a service layer between all controller and Repositories.  Please create a User Story to refactor this throughout the app and we will keep it in the back log for now until will prioritize it.  Going forward please create service layer and use as described.